.github/workflows/spelling.yaml

```diff
@@ -15,5 +15,5 @@ jobs:
     - name: Check Spelling
       uses: crate-ci/typos@7ad296c72fa8265059cc03d1eda562fbdfcd6df2 # v1.9.0
```

Comment:
v1.9.0 is a very old release, maybe we need an upgrade.
https://github.com/crate-ci/typos/releases